lib/libzfs/libzfs_util.c

```diff
@@ -59,6 +59,31 @@ libzfs_errno(libzfs_handle_t *hdl)
 }
 
 const char *
+libzfs_error_init(int error)
```

Comment:
The name `libzfs_error_init` makes me think you're initializing some `libzfs_error` subsystem.